From 31863361dc58ea9c0c0dafc72aa16941d302ac5b Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Thu, 10 Jul 2014 01:04:35 -0400
Subject: [base] remove unneeded variable

Message-id: <20140710010434.22146.88.stgit@gir.stowe>
Patchwork-id: 85600
O-Subject: [RHEL7.1 PATCH 061/105] drivers: base: remove unneeded variable
Bugzilla: 1114228
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1114228
Upstream Status: 879f1bec8e136c7bc71e38715e62a73b75f91d78

commit 879f1bec8e136c7bc71e38715e62a73b75f91d78
Author: Seth Jennings <sjenning@linux.vnet.ibm.com>
Date:   Tue Aug 20 12:12:58 2013 -0500

    drivers: base: remove unneeded variable

    The error variable is not needed.

    Signed-off-by: Seth Jennings <sjenning@linux.vnet.ibm.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/base/memory.c b/drivers/base/memory.c
index cecd660..f6d11f8 100644
--- a/drivers/base/memory.c
+++ b/drivers/base/memory.c
@@ -567,16 +567,13 @@ static const struct attribute_group *memory_memblk_attr_groups[] = {
 static
 int register_memory(struct memory_block *memory)
 {
- int error;
-
  memory->dev.bus = &memory_subsys;
  memory->dev.id = memory->start_section_nr / sections_per_block;
  memory->dev.release = memory_block_release;
  memory->dev.groups = memory_memblk_attr_groups;
  memory->dev.offline = memory->state == MEM_OFFLINE;
 
- error = device_register(&memory->dev);
- return error;
+ return device_register(&memory->dev);
 }
 
 static int init_memory_block(struct memory_block **memory,
-- 
1.7.1